conductr_cli/license.py

```diff
@@ -7,6 +7,11 @@
 
 EXPIRY_DATE_DISPLAY_FORMAT = '%a %d %b %Y %H:%M%p'
 
+UNLICENSED_DISPLAY_TEXT = 'UNLICENSED - please use "conduct load-license" to use more than one agent. ' \
+                          'Additional agents are available for free for registered users.\n' \
```

Comment:
@huntc - I've updated the text to:

```
UNLICENSED - please use "conduct load-license" to use more than one agent. Additional agents are freely available for registered users
Max ConductR agents: 1
Grants: conductr, cinnamon, akka-sbr
```

I'm going to merge this PR, and if you'd like to change this text further - I'm happy to do so.